backend/src/main/java/org/fungover/zipp/security/SecurityConfig.java

```diff
@@ -18,14 +19,18 @@ public SecurityConfig(CustomOAuth2UserService co2us) {
 
     @Bean
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
-        http.csrf(csrf -> csrf.disable()).authorizeHttpRequests(auth -> auth
-                .requestMatchers("/", "/login", "/favicon.ico", "/favicon/**", "/css/**", "/images/**", "/js/**")
-                .permitAll().anyRequest().authenticated())
+
+        http.authorizeHttpRequests(
+                auth -> auth.requestMatchers("/", "/login", "/login/**", "/favicon.ico", "/favicon/**", "/css/**",
+                        "/js/**", "/images/**", "/img/**", "/webauthn/**").permitAll().anyRequest().authenticated())
                 .oauth2Login(oauth2 -> oauth2.loginPage("/login").defaultSuccessUrl("/", true)
                         .userInfoEndpoint(userInfo -> userInfo.userService(co2us)))
+                .webAuthn(webauthn -> webauthn.rpId("localhost").allowedOrigins("http://localhost:8080"))
```

Comment:
Maybe we need to use some properties or env here for it to work in production? Can't be localhost when published on zipp.city?